test_tools/src/test_tools/doctest_unittest.py

```diff
@@ -0,0 +1,36 @@
+#! /usr/bin/env python
+
+import doctest
+import importlib
+import os
+import rospkg
+import unittest
+
+
+class _TestDocTests(unittest.TestCase):
```

Comment:
I don't think this should be private if you intend to import this from other files/modules/packages which you do). Why did you choose to do so?

Reply:
To prevent nosetest from running this class in the test files which import this class